products/idn/api/patch-guide.md

```diff
@@ -0,0 +1,474 @@
+---
+id: patch-guide
+title: Patch Guide
+pagination_label: Patch Guide
+sidebar_label: Patch Guide
+sidebar_position: 8
+sidebar_class_name: patchGuide
+keywords: ['patch']
+description: Read this guide to learn how to send PATCH requests. 
+tags: ['patch']
+---
+
+## PATCH requests 
+
+You can use the IdentityNow (IDN) APIs to update resources. Many of the APIs offer multiple ways to do so: 
```

Comment:
You can use the IdentityNow (IDN) APIs to update **existing** resources? Or is that implied as well?

Reply:
I'll add it - I think it might be implied, but it's important enough to make a bit redundant.